samples/subsys/logging/syst/src/main.c

```diff
@@ -58,6 +60,10 @@ void main(void)
 	LOG_DBG("Debug message example, %d, %d, %d", 1, 2, 3);
 	LOG_DBG("Debug message example, %d, %d, %d, 0x%x", 1, 2, 3, 4);
 
+	size_t log_type = LOG_OUTPUT_TEXT;
+
+	log_backend_format_set(log_backend_get_by_name("log_backend_uart"), log_type);
```

Comment:
I would put all existing log messages in another function and then use first format )default) to display the, switch, display them again in the same format, switch back and display the original format, this way we do not lose functionality of the sample originally designed for demonstrating sys-t

Reply:
@nashif Sure. Have updated the main.c for sys-t sample as suggested. Please re-review.